### DISCUSSION TITLE
Bonsai 2.8 - IronPython Python Transform editor not opening

### DISCUSSION THREAD

**Discussion by vncntprvst** [bonsai-rx/bonsai#1929] [Q&A] [answered]
Hi,
I'm updating some workflows from 2.6 to 2.8. The `Python Transform` operator from the IronPython library had to be updated, so I'm copy-pasting the former Python code to the new operator. Strangely, the editor does not pop up when double-clicking on the operator. Have others observed that behavior? The Script field does not have a drop-down menu to visualize the script either, so it's not convenient to write code in there. 
Thanks

**Comment by bruno-f-cruz** [comment, marked as answer]
Have you installed the corresponding Design package?

**Reply by vncntprvst** [reply]
Ha, no. My bad. Forgot about this :) 
It works now.
Thank you Bruno!
